packages/db/schema/mysql.ts

```diff
@@ -333,6 +333,7 @@ export const sections = mysqlTable("section", {
   width: int(),
   height: int(),
   name: text(),
+  options: text().default('{"json": {}}').notNull(), // empty superjson object
```

Comment:
Make it nullable (IMO category / empty section should have options can be null for now) (Same for sqlite)